platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegionDefinition.java

```diff
@@ -0,0 +1,65 @@
+package com.mapbox.mapboxsdk.offline;
+
+import com.mapbox.mapboxsdk.geometry.LatLngBounds;
+
+/**
+ * An offline region defined by a style URL, geographic bounding box, zoom range, and
+ * device pixel ratio.
+ *
+ * Both minZoom and maxZoom must be ≥ 0, and maxZoom must be ≥ minZoom.
+ *
+ * maxZoom may be ∞, in which case for each tile source, the region will include
+ * tiles from minZoom up to the maximum zoom level provided by that source.
+ *
+ * pixelRatio must be ≥ 0 and should typically be 1.0 or 2.0.
+ */
+public class OfflineRegionDefinition {
```

Comment:
Let's structure the Java class definitions so that we can introduce new region definition types in the future without breaking code that creates the current type of region definition.

```
public interface OfflineRegionDefinition {}

class LatLngBoundsOfflineRegionDefinition implements OfflineRegionDefinition {
  ...
}
```